### PR TITLE
Fix scikit bio diversity

### PR DESCRIPTION
**Pull request by mvdbeek**
FOR CONTRIBUTOR:
* [ ] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [ ] - License permits unrestricted use (educational + commercial)
* [ ] - This PR adds a new tool or tool collection
* [x] - This PR updates an existing tool or tool collection
* [ ] - This PR does something else (explain below)

### CODE REVIEW

File: data_managers/data_manager_snpsift_dbnsfp/data_manager/data_manager_snpsift_dbnsfp.py [hunk 1]
```diff
@@ -119,7 +120,7 @@ def download_dbnsfp_database(url, output_file):
                         tfh.close()
                         tempfiles.append(file + "_%d" % len(tempfiles))
                         tfh = open(tempfiles[-1], 'w')
-                        print >> sys.stderr, "%s [%d] pos: %d < %d" % (file, i, pos, lastpos)
+                        print( "%s [%d] pos: %d < %d" % (file, i, pos, lastpos), file=sys.stderr)
```
Comment (nsoranzo) [inline data_managers/data_manager_snpsift_dbnsfp/data_manager/data_manager_snpsift_dbnsfp.py]:
```suggestion
                        print("%s [%d] pos: %d < %d" % (file, i, pos, lastpos), file=sys.stderr)
```

File: tools/scikit-bio/scikit_bio_diversity_beta_diversity.py [hunk 2]
```diff
@@ -42,47 +43,46 @@ def __main__():
...
+        columns = list(map(lambda x: int(x) - 1, options.sample_columns.split(",")))
 
-    max_col = max( columns + [otu_column] )
-    counts = [ [] for x in columns ]
+    max_col = max(columns + [otu_column])
+    counts = [[] for x in columns]
     sample_names = []
     otu_names = []
-    with open( options.input, 'rb' ) as fh:
+    with codecs.open(options.input, 'r', encoding="utf-8") as fh:
```
Comment (nsoranzo) [inline tools/scikit-bio/scikit_bio_diversity_beta_diversity.py]:
```suggestion
    with io.open(options.input, 'r', encoding="utf-8") as fh:
```

File: tools/scikit-bio/scikit_bio_diversity_beta_diversity.py [hunk 3]
```diff
@@ -42,47 +43,46 @@ def __main__():
...
     extra_kwds = {}
     if options.distance_metric in NEEDS_OTU_NAMES:
         extra_kwds['otu_ids'] = otu_names
     if options.distance_metric in NEEDS_TREE:
-        assert options.tree, Exception( "You must provide a newick tree when using '%s'" % options.distance_metric )
-        # NB: TreeNode apparently needs unicode files
-        with codecs.open( options.tree, 'rb', 'utf-8' ) as fh:
-            extra_kwds['tree'] = TreeNode.read( fh )
+        assert options.tree, Exception("You must provide a newick tree when using '%s'" % options.distance_metric)
+        with codecs.open(options.tree, 'r', 'utf-8') as fh:
```
Comment (nsoranzo) [inline tools/scikit-bio/scikit_bio_diversity_beta_diversity.py]:
```suggestion
        with io.open(options.tree, 'r', 'utf-8') as fh:
```

**Comment by bgruening** [comment]
mh ... https://travis-ci.org/galaxyproject/tools-iuc/jobs/642024026?utm_medium=notification&utm_source=email

**Comment by nsoranzo** [comment]
The upload of `data_manager_fetch_gene_annotation` to the Test Tool Shed fails because the owner on `.shed.yml` is scottx611x , but there is no corresponding repo on the TTS and we cannot create one ourselves.

**Pull request merged**